src/logic/safe/store/models/types/transaction.ts

```diff
@@ -17,7 +17,7 @@ export type TxArgs = {
   refundReceiver: string
   safeInstance: GnosisSafe
   safeTxGas: string
-  sender?: string
+  sender: string
```

Comment:
Why is this now required?

Reply:
Because we were setting it to required by using `intersection types` to make it required